package.json

```diff
@@ -144,12 +144,14 @@
     "markdown-it-deflist": "^2.0.3",
     "markdown-it-footnote": "^3.0.1",
     "markdown-it-gridtables": "^0.3.0",
+    "markdown-it-highlightjs": "^4.2.0",
     "markdown-it-implicit-figures": "^0.9.0",
-    "markdown-it-pandoc": "^2.1.0",
+    "markdown-it-mathjax3": "^4.3.2",
+    "markdown-it-pandoc": "file:../markdown-it-pandoc",
```

Comment:
This will obviously change to ^2.3.0 after markdown-it-pandoc side is merged/released.